### PR TITLE
[RPA-531] Switch to puppeteer for pdf's

### PR DESCRIPTION
**Pull request by FranekMarciniak**

### CODE REVIEW

File: runbotics/runbotics-desktop/src/actions/browser/BrowserAutomation.ts [hunk 1]
```diff
@@ -281,47 +281,28 @@ class BrowserAutomation extends StatefulActionHandler {
...
-                });
-                await page.pdf({
-                    path: `${fileName}.pdf`,
-                    format: 'A4',
+        try {
+            generatePdf(target, {})
+                .then(pdfBuffer => {
+                    writeFile(`${fileName}.pdf`, Buffer.from(pdfBuffer), 'binary', (err) => {
+                        if (err) {
+                            console.log("There was an error writing the pdf")
```
Comment (preludium) [inline runbotics/runbotics-desktop/src/actions/browser/BrowserAutomation.ts]: use custom logger + this should be `this.logger.error`

Comment (preludium) [reply runbotics/runbotics-desktop/src/actions/browser/BrowserAutomation.ts]: it won't go to catch unless you throw error here imo

File: runbotics/runbotics-orchestrator-ui/src/views/process/ProcessBuildView/Modeler/ConfigureActionPanel/Actions/browser.actions.ts [hunk 2]
```diff
@@ -681,6 +681,7 @@ const getBrowserActions: () => Record<string, IBpmnAction> = () => ({
         translateKey: 'Process.Details.Modeler.Actions.Browser.PrintToPdf.Label',
         script: 'browser.selenium.printToPdf',
         runner: Runner.DESKTOP_SCRIPT,
+        system: ActionSystem.WINDOWS,
```
Comment (preludium) [inline runbotics/runbotics-orchestrator-ui/src/views/process/ProcessBuildView/Modeler/ConfigureActionPanel/Actions/browser.actions.ts:722]: so this only works on windows for now? `getChromiumPath` is kinda tricky

**Review by preludium — DISMISSED** [review]
adjust version number

File: runbotics/runbotics-orchestrator-ui/src/views/process/ProcessBuildView/Modeler/ConfigureActionPanel/Actions/browser.actions.ts (hunk 2 above)
Comment (FranekMarciniak) [reply runbotics/runbotics-orchestrator-ui/src/views/process/ProcessBuildView/Modeler/ConfigureActionPanel/Actions/browser.actions.ts:722]: yup, if we can get chrome on linux we can refactor this

**Review by preludium — DISMISSED** [review]

**Review by preludium — DISMISSED** [review]

**Review by preludium — APPROVED** [review]

**Pull request merged**